cmake/configs/base.cmake

```diff
@@ -287,6 +287,12 @@ config_bool(
     DEFAULT OFF
 )
 
+config_bool(
+    ENABLE_GCP
+    "Build the GCP (Google Cloud Platform) storage extension"
```

Comment:
Let's choose one or the other, it's a bit overkill to have both.  "Build the Google Cloud Platform storage extension" is fine.

Reply:
addressed here: [e7cd853](https://github.com/wiredtiger/wiredtiger/pull/8688/commits/e7cd853492f5b96a6798679c3db140b75ce568eb)